frontend/ui/widget/inputtext.lua

```diff
@@ -166,28 +166,31 @@ if Device:isTouchDevice() or Device:hasDPad() then
                         return true
                     end
                 end
-                local input_dialog
-                input_dialog = require("ui/widget/inputdialog"):new{
-                    title = _("Clipboard"),
+                local clipboard_value = Device.input.getClipboardText()
+                local clipboard_dialog
+                clipboard_dialog = require("ui/widget/textviewer"):new{
+                    title = (clipboard_value == nil or clipboard_value == "") and _("Clipboard (empty)") or _("Clipboard"),
+                    text = clipboard_value,
+                    width = math.floor(Screen:getWidth() * 0.8),
+                    height = math.floor(Screen:getHeight() * 0.39),
```

Comment:
Do we use this value elsewhere?

Reply:
No, it is designed to look nice and show 5 lines of the clipboard without scrolling for my screen (PW2).
Hope it will fit to other screen resolutions.

Reply:
![2](https://user-images.githubusercontent.com/62179190/123595941-37080380-d7fa-11eb-9a95-48a8ca69650b.png)

Reply:
"Enter page number" dialog is transparent just for screenshot making only, it will not be.

Reply:
**height factor 0.39**
![3](https://user-images.githubusercontent.com/62179190/123619223-a2a99b00-d811-11eb-90bd-3c84deb6f069.png)

**height factor 0.4**
![4](https://user-images.githubusercontent.com/62179190/123619246-a9381280-d811-11eb-9dc0-b2eab82b880b.png)

Reply:
0.4 would look round enough so nobody will wonder why 0.4 - unlike 0.39 :)
Also, what look nice on your device will probably look different if you choose another DPI, and on other devices with a different resolution.

Reply:
Fine, 0.4 agreed.